### ISSUE TITLE
Error in browser console after visiting localhost:5000

### ISSUE THREAD

**Issue by malisetti** [bpanel-org/bpanel#190]
Unhandled promise rejection TypeError: "can't convert undefined to object"

rootReducer rootReducer.js:46 persistReducer2Redux

l index.js:13 E index.js:13 _invoke index.js:13 n index.js:13 o index.js:13 u index.js:13 s index.js:13 P index.js:13 s index.js:13 exports index.js:13 effectsMiddleware index.js:13 nextMiddleware plugins.js:229 node_modules index.js:80 nextMiddleware plugins.js:228 pluginMiddleware plugins.js:230 createThunkMiddleware Redux	clientMiddleware middleware.js:44 errorCatcherMiddleware middleware.js:14 dispatch Redux	clientMiddleware middleware.js:42 errorCatcherMiddleware middleware.js:14 dispatch Redux

_callee3$ clientActions.js:77 E index.js:13 _invoke index.js:13 n index.js:13 step clientActions.js:19 step clientActions.js:19 s index.js:13 N index.js:13 c index.js:13 

index.js:13:80952

n index.js:13 exports index.js:13 j index.js:13 exports index.js:13 <anonymous> index.js:13 m index.js:13 w index.js:13

**Comment by bucko13** [comment]
Thanks for submitting this! I haven't seen this error before but to help with the debugging would you mind sharing some information about your setup?

- [ ] versions of node and npm
- [ ] version of bPanel
- [ ] what plugins you have installed (you can use `bpanel-cli list` to check this if you have the cli tool installed and are loading using the default config directory `~/.bpanel`

**Comment by malisetti** [comment]
1. Node version v11.1.0, NPM version 6.8.0
2. bPanel is built from development branch, commit dc6f4d8bb7a552958c6ee3c59b533e24702f2f17
3. config.js content
```
module.exports = {
  plugins: ['@bpanel/genesis-theme', '@bpanel/settings', '@bpanel/connection-manager'],
  localPlugins: [],
}
```

**Comment by bucko13** [comment]
Thanks for catching that! Fixed in cf0f3ce.
